manifests/bucketeer/charts/experiment-calculator/templates/envoy-configmap.yaml

```diff
@@ -0,0 +1,235 @@
+                        typed_config:
+                          '@type': type.googleapis.com/envoy.extensions.filters.http.health_check.v3.HealthCheck
+                          cluster_min_healthy_percentages:
+                            calculator:
+                              value: 25
+                          headers:
+                            - name: :path
+                              string_match:
+                                exact: /health
+                          pass_through_mode: false
+                      - name: envoy.filters.http.router
+                    route_config:
+                      virtual_hosts:
+                        - domains:
+                            - '*'
+                          name: ingress_services
+                          routes:
+                            - match:
+                                headers:
+                                  - name: content-type
+                                    string_match:
+                                      exact: application/grpc
+                                prefix: /
+                              route:
+                                cluster: calculator
+                                retry_policy:
+                                  num_retries: 3
+                                  retry_on: 5xx
+                                timeout: 15s
+                    stat_prefix: ingress_http
+                    stream_idle_timeout: 300s
+              transport_socket:
+                name: envoy.transport_sockets.tls
+                typed_config:
+                  '@type': type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.DownstreamTlsContext
+                  common_tls_context:
+                    alpn_protocols:
+                      - h2
+                    tls_certificates:
+                      - certificate_chain:
+                          filename: /usr/local/certs/service/tls.crt
+                        private_key:
+                          filename: /usr/local/certs/service/tls.key
+                  require_client_certificate: true
+        - name: egress
+          address:
+            socket_address:
+              address: 127.0.0.1
+              port_value: 9001
+          filter_chains:
+            - filters:
+                - name: envoy.filters.network.http_connection_manager
+                  typed_config:
+                    '@type': type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+                    access_log:
+                      name: envoy.access_loggers.file
+                      typed_config:
+                        '@type': type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+                        path: /dev/stdout
+                    codec_type: auto
+                    http_filters:
+                      - name: envoy.filters.http.router
```

Comment:
Please add the `typed_config`.

https://github.com/bucketeer-io/bucketeer/pull/459/files#diff-6c6dfb590e04fdd74ea9cf9c5e456da2a4cea085da6a192c92dc0e67cf6ed317R181-R182